src/platform/forms-system/src/js/components/ContactInfoReview.jsx

```diff
@@ -12,10 +12,13 @@ import {
   CONTACT_EDIT,
   renderTelephone,
   contactInfoPropTypes,
+  validateEmail,
+  validatePhone,
+  validateZipcode,
 } from '../utilities/data/profile';
 
 /**
- * Contact info for review & submit page
+ * Contact info fields shown on the review & submit page
  * @param {Object} data - form data
  * @param {function} editPage - edit page callback
  * @param {import('../utilities/data/profile').ContactInfoContent} content
```

Comment:
Doesn't have to be addressed in this PR, but does this need to be passed in as argument? Or can we just import it within this file?